Update intro to OpenAPI on minimal APIs section (#27107)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/minimal-apis/openapi.md

:::moniker range=">= aspnetcore-7.0"

> The [OpenAPI specification](https://spec.openapis.org/oas/latest.html)) is a programming language-agnostic standard for documenting HTTP APIs. This standard is supported in minimal APIs
> through a combination of built-in APIs and open-source libraries. There are three key aspects to OpenAPI integration in your application:

Reviewer: ```suggestion
The [OpenAPI specification](https://spec.openapis.org/oas/latest.html)) is a programming language-agnostic standard for documenting HTTP APIs. This standard is supported in minimal APIs through a combination of built-in APIs and open-source libraries. There are three key aspects to OpenAPI integration in an application:
```

> * Generating information about the endpoints in your application

Reviewer: ```suggestion
* Generating information about the endpoints in the app.
```

> * Gathering the information into a format that matches the OpenAPI schema
> * Exposing the generated OpenAPI schema via a visual UI or a serialized file

Reviewer: ```suggestion
* Gathering the information into a format that matches the OpenAPI schema.
* Exposing the generated OpenAPI schema via a visual UI or a serialized file.
```

> Minimal APIs provides built-in support for generating information about endpoint in an application via the `Microsoft.AspNetCore.OpenApi` package. For exposing the generated OpenAPI
> definition via a visual UI, you will need to integrate a third-party dependency. 

Reviewer: ```suggestion
Minimal APIs provides built-in support for generating information about endpoints in an app via the `Microsoft.AspNetCore.OpenApi` package. Exposing the generated OpenAPI definition via a visual UI requires a third-party package. 
```
